ttnn/cpp/ttnn/operations/downsample/device/downsample_op.hpp

```diff
@@ -25,6 +28,21 @@ struct Downsample {
     const auto attribute_values() const {
         return std::make_tuple(std::cref(this->downsample_params), std::cref(this->output_dtype));
     }
+    static ttnn::Tensor execute_on_worker_thread(
+        const Tensor& input_tensor_a, std::array<uint32_t, 5> downsample_params, std::optional<DataType> output_dtype) {
+        std::vector<Tensor> output_tensors = {Tensor(operation::get_workers_for_op_output({input_tensor_a}))};
+        operation::launch_op(
```

Comment:
This should not use `launch_op`.

Please switch to `operation::run`

Reply:
Done.